src/Sylius/Behat/Context/Transform/ProductContext.php

```diff
@@ -35,6 +35,8 @@ public function __construct(ProductRepositoryInterface $productRepository)
     /**
      * @Transform /^product "([^"]+)"$/
      * @Transform /^"([^"]+)" product$/
+     * @Transform /^"([^"]+)" products$/
+     * @Transform /^(?:a|an) "([^"]+)"$/
      * @Transform :product
```

Comment:
Unfortunately not, because I have a/an preposition before products, so I need a regex